### PR TITLE
Change PostgreSQL version from 9.3 to 9.5

### PR DESCRIPTION
**Pull request by justinboisvert**
Rong recently told me our landscape servers use version 9.5 instead of 9.3 (the version at the time from when we were development) This updates the path folder to the latest version so it can work again but it would be nice to have a small yaml file turner reads from that would contain the version that the user could simply edit.

### CODE REVIEW

**Comment by unixorn** [comment]
Maybe set the PostgreSQL version as a trollop command line option and make the default value 9.5?

It would be nice to make all the paths command line options with reasonable defaults so the user can run it with different paths without having to massage a yaml file.

**Comment by justinboisvert** [comment]
Joe, I just realized that Landscape-Turner already supports this. You can override the postgresql specific path through -o oldpath=newpath. Looks like trollop had us covered.

File: lib/landscape-turner/backup.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def self.backup_with_cli_args(args = ARGV)
       :x509_certificate  => "/etc/ssl/certs/landscape_server.pem",
       :ca_certificate    => "/etc/ssl/certs/landscape_server_ca.crt",
       :ssl_private_key   => "/etc/ssl/private/landscape_server.key",
-      :postgresql_config => "/etc/postgresql/9.3/main",
+      :postgresql_config => "/etc/postgresql/9.5/main",
```
Comment (unixorn) [inline lib/landscape-turner/backup.rb]: @justinboisvert, what I meant was to take all these default arguments to `self.backup_with_cli_args` and add them as trollop options with defaults as listed there.

That way `--help` would expose all these settings and display what the default values are.

Comment (justinboisvert) [inline lib/landscape-turner/backup.rb]: Ah, I see what you mean.

I feel like having a trollop option for each file path would be a bit overkill. How about having just an option that would list all of the default paths? That way if someone would like to use a different path for something, they can use the override feature already implemented.

Comment (unixorn) [inline lib/landscape-turner/backup.rb]: Using `--override` makes them have to go through the code to find out what the paths are. Having trollop options with default values lets them just use `--help` to see what paths are available to set, along with what we're expecting to see.

Comment (justinboisvert) [inline lib/landscape-turner/backup.rb]: That's why I'm saying we could have an option where turner lists all of the paths that are being backed up. That can be documented on the Github page and that way the user can infer which path to override with a new one.

Comment (unixorn) [inline lib/landscape-turner/backup.rb]: That's an overly complex workaround. First they'd have to use `--help` to figure out how to dump the available paths, then they'd have to figure out what the current defaults are, then they'd have to `--override` all of them. Making the paths each an option means trollop will spit them all out, with their default values, when they use `--help`.

Comment (justinboisvert) [inline lib/landscape-turner/backup.rb]: OK - fair enough.

File: lib/landscape-turner/backup.rb [hunk 2]
```diff
@@ -43,13 +43,13 @@ def self.backup_with_cli_args(args = ARGV)
...
     paths.each { |k, v| paths[k] = "#{prefix}#{v}" }
 
-    (new_args[:override] || []).each do |o|
-      key, val = o.split("=")
-      key = key.to_sym
-      raise "Unrecognized name argument: #{key.inspect}!" unless paths[key]
-      paths[key] = val
-    end
-
+   path_args = new_args.clone
```
Comment (unixorn) [inline lib/landscape-turner/backup.rb]: These aren't trollop opts, so trollop won't list them in `--help`.

**Pull request merged**